### DISCUSSION TITLE
iOS SDK -> Conference

### DISCUSSION THREAD

**Discussion by ivoperic** [ant-media/Ant-Media-Server#6104] [Q&A] [answered]
iOS SDK doesn't work with current Conference implementation.

Tried in Sample App from github
https://github.com/ant-media/WebRTC-iOS-SDK

`swift
        self.conferenceClient?.joinRoom(roomId: roomId)
`

Publishing works, but there is no callback for newVideoTrack
Same configuration works on latest android SDK

**Comment by USAMAWIZARD** [comment]
let me check and get back to you on this , which Ant Media Version are you using right now ? there are some know issues right now let me update you soon

**Comment by USAMAWIZARD** [comment, marked as answer]
@ivoperic there has been some new fixes on conference example can you please try it out now and let me know if it works for you.
https://github.com/ant-media/WebRTC-iOS-SDK/

**Comment by ivoperic** [comment]
thanks
